### DISCUSSION TITLE
When next release (2.7.8?)?

### DISCUSSION THREAD

**Discussion by dmindfuse** [keepassxreboot/keepassxc#10668] [Q&A] [answered]
Hi. Wanting to migratie from 1Password to keepassxc I am waiting for https://github.com/keepassxreboot/keepassxc/pull/10499 to be made available in the next release. Reason being TOTP imports do not work in 2.7.7 for .1Pux files. Hope it’s not going to be months? Anyone know?

**Comment by droidmonkey** [comment, marked as answer]
Today. You can always use a snapshot build for one time actions like imports as well. Https://snapshot.keepassxc.org
